docs_src/microservices/application/GeneralAppServiceConfig.md

```diff
@@ -108,6 +108,27 @@ Only used for `messagebus` binding type
 | MessageBus SubscribeHost `Protocol` | tcp** | Indicates the protocol number for publishing to the Message Bus |
 | **MessageBus Optional** | ... | This section is used for optional configuration specific to the Message Bus type used. Please refer to [go-mod-messaging](https://github.com/edgexfoundry/go-mod-messaging/blob/master/README.md) for more details |
 
+### MqttBrokerConfig
+
+This section defines the external MQTT Broker connect information.
+Only used for `external-mqtt` trigger binding type
+
+!!! note
```

Comment:
Your note is not coming through here.  You need to indent the note text under !!! note